src/generic_ui/polymer/contact.ts

```diff
@@ -16,12 +16,15 @@ Polymer({
     this.model = model;
     this.GettingConsentState = UI.GettingConsentState;
     this.SharingConsentState = UI.SharingConsentState;
+    this.instanceTryingToGet = false;
```

Comment:
can this be isInstanceTryingToGet or just isTryingToGet?  The name "instanceTryingToGet" makes it sound like an Instance object, not a boolean